x-pack/plugins/security_solution/public/common/components/drag_and_drop/draggable_wrapper.tsx

```diff
@@ -370,7 +374,7 @@ const DraggableWrapperComponent: React.FC<Props> = ({
       <WithHoverActions
         alwaysShow={showTopN || hoverActionsOwnFocus}
         closePopOverTrigger={closePopOverTrigger}
-        hoverContent={hoverContent}
+        hoverContent={disableHoverActions(timelineId) ? undefined : hoverContent}
```

Comment:
You could actually put this with the function above in a `useMemo` that just returns hoverContent?